universal_transfer_operator/example_dags/example_universal_transfer_operator.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+
+from airflow import DAG
+from datasets.file import File
+from datasets.table import Table
+from universal_transfer_operator import UniversalTransferOperator
+
+from constants import TransferMode
+
+START_DATE = datetime(2000, 1, 1)
+with DAG(
+    "example_universal_transfer_operator",
+    schedule_interval=None,
+    start_date=START_DATE,
+    catchup=False,
+) as dag:
+    transfer_non_native = UniversalTransferOperator(
+        task_id="transfer_non_native",
+        source_dataset=File("gs://uto-test/uto/", conn_id="google_cloud_default", extra={}),
+        destination_dataset=File("s3://astro-sdk-test/uto/", conn_id="aws_default", extra={}),
+    )
+
+    transfer_fivetran_with_connector_id = UniversalTransferOperator(
+        task_id="transfer_fivetran_with_connector_id",
+        source_dataset=File("s3://astro-sdk-test/uto/", conn_id="aws_default", extra={}),
```

Comment:
Since `extra` param in this entire file is empty, can we just not pass it, please

Reply:
```suggestion
        source_dataset=File("s3://astro-sdk-test/uto/", conn_id="aws_default"),
```

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1492/commits/831d627d1a86acab300968fc56d11bc7f016b4b0)